### ISSUE TITLE
I would like to determine a naming convention for restricting Binding type.

### ISSUE THREAD

**Issue by Yuutakasan** [Apicurio/apicurio-data-models#569]
For example, in the case of Kafka's Server Binding

servers:
  production:
    bindings:
      kafka:
        schemaRegistryUrl: 'https://my-schema-registry.com'
        schemaRegistryVendor: 'confluent'
        bindingVersion: '0.4.0'

but currently there is only one Binding type and it is defined as follows

  - name: Binding
    traits:
      - Extensible
    properties:
      - name: '*'
        type: any
    propertyOrder:
      - $this
      - $Extensible

No restrictions.

I would like to add individual restrictions, but is there a naming convention here?

**Comment by EricWittmann** [comment]
Yes bindings do need a bit more work (modeling), for sure.  I think the naming convention should be driven by the text of the AsyncAPI specification.  For example, the spec has a table of Operation Bindings like this:

![image](https://user-images.githubusercontent.com/1890703/220745197-916aacbc-e0d8-4c56-ba1c-bb8bf86f0a79.png)

So I think the convention should be to remove non-alphanumeric characters from the value in the Type column of that table.  So in this case, it would be:

- HTTPOperationBinding
- WebSocketsOperationBinding
- KafkaOperationBinding
- AnypointMQOperationBinding
- AMQPOperationBinding
- AMQP10OperationBinding

And so forth.

I believe this is consistent with your suggestion, so I agree.  :)

**Comment by Yuutakasan** [comment]
@EricWittmann 
Sorry for the delay. I will try to work with our team on the Binding extension.
